### PR TITLE
Support Dark mode for Share UI feature

### PR DESCRIPTION
**Pull request by plateaukao**

### CODE REVIEW

File: app/src/main/java/com/linecorp/linesdktest/MainActivity.java [hunk 1]
```diff
@@ -47,10 +47,12 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
         getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_ALWAYS_HIDDEN);
 
-        getSupportFragmentManager()
-                .beginTransaction()
-                .add(R.id.container, new MenuFragment())
-                .commit();
+        if (getSupportFragmentManager().getFragments().isEmpty()) {
```
Comment (plateaukao) [inline app/src/main/java/com/linecorp/linesdktest/MainActivity.java]: When Activity is re-created, and fragment manager is not empty, there's no need to create a new MenuFragment.

File: line-sdk/src/main/res/values-night/colors.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <color name="target_item_text">@android:color/primary_text_dark</color>
+    <color name="target_item_normal">@android:color/background_dark</color>
+    <color name="target_item_pressed">@android:color/darker_gray</color>
+    <color name="target_item_background_pressed">#1e1e1e</color>
+
+    <color name="text_highlight">#07bf3f</color>
+
+    <color name="header_text">@android:color/primary_text_dark</color>
+    <color name="primary_text">@android:color/primary_text_light</color>
+    <color name="background">@android:color/background_dark</color>
```
Comment (tsairene) [inline line-sdk/src/main/res/values-night/colors.xml]: missing `share_button_text` color?

File: line-sdk/src/main/res/values-night/colors.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+    <color name="target_item_text">@android:color/primary_text_dark</color>
+    <color name="target_item_normal">@android:color/background_dark</color>
+    <color name="target_item_pressed">@android:color/darker_gray</color>
+    <color name="target_item_background_pressed">#1e1e1e</color>
+
+    <color name="text_highlight">#07bf3f</color>
+
+    <color name="header_text">@android:color/primary_text_dark</color>
```
Comment (tsairene) [inline line-sdk/src/main/res/values-night/colors.xml]: should be `primary_text_light`?

File: line-sdk/src/main/res/values-night/colors.xml (hunk 2 above)
Comment (plateaukao) [reply line-sdk/src/main/res/values-night/colors.xml]: For the share button text, it's not changed; since the button color is still green.

Comment (tsairene) [reply line-sdk/src/main/res/values-night/colors.xml]: Got it~~

**Pull request merged**